platform/darwin/src/MGLMultiPoint.mm

```diff
@@ -93,6 +98,45 @@ - (void)getCoordinates:(CLLocationCoordinate2D *)coords range:(NSRange)range
     }
 }
 
+- (void)setCoordinates:(CLLocationCoordinate2D *)coords range:(NSRange)range
+{
+    if ([self isMemberOfClass:[MGLMultiPoint class]])
+    {
+        [[NSException exceptionWithName:@"MGLAbstractClassException"
+                                 reason:@"MGLMultiPoint is an abstract class"
+                               userInfo:nil] raise];
+    }
+
+    NSAssert(range.length > 0, @"Empty coordinate range");
+
+    [self willChangeValueForKey:@"coordinates"];
+    if (range.location >= _count)
```

Comment:
Let me know how 2d46f20d8b0713a35d7fa97729fb5db285cb098a looks @1ec5. I think what I did there is kosher.